test/13_arraycommon.jl

```diff
@@ -1426,8 +1426,8 @@ end
         @test sprint((io,a)->show(io, "text/plain", a), x) ==
             """
             3-element $CategoricalArray{$(Union{Missing,Int}),1,UInt32}:
-             2
-             1
+             2      
```

Comment:
Unintented whitespace?

Reply:
It needs to be there for tests to pass on earlier Julia versions... I accidentally removed this whitespace when I saved the file in VSCode with automatic whitespace trimming turned on.